modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/internal/model/listener/UserModelListener.java

```diff
@@ -49,13 +51,27 @@ public void onAfterUpdate(User originalUser, User user)
 				return;
 			}
 
+			long groupId = 0;
+
+			try {
+				groupId = user.getGroupId();
+			}
+			catch (Exception exception) {
+				_log.error(
+					"User " + user.getUserId() +
+						" is not related to any groups");
+				_log.error(exception.toString());
+
+				return;
+			}
```

Comment:
Hey @rijema thanks for your changes!

I'm not sure if log this error will be relevant, let me know my suggestion below makes sense:

```suggestion
			Group group = user.getGroup();
			
			long groupId = (group == null) ? 0 : group.getGroupId();
```

Reply:
@rijema the `groupId` is used for getAvailableLocales, if it's 0 it will get the company's available locales otherwise it will get the locales base on the groupId, that is why I believe we don't need to log this error.